.devcontainer/devcontainer.json

```diff
@@ -6,10 +6,17 @@
   ],
   "service": "client",
   "workspaceFolder": "/app",
-  "extensions": [
-    "ms-azuretools.vscode-docker",
-    "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
-    "ms-dotnettools.csharp"
-  ]
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-azuretools.vscode-docker",
+        "dbaeumer.vscode-eslint",
+        "esbenp.prettier-vscode",
+        "ms-dotnettools.csharp",
+        "github.vscode-github-actions",
+        "github.copilot-chat",
+        "github.copilot"
+      ]
+    }
+  }
```

Comment:
Q: Brauchen wir die devcontainer noch?
PP: In separatem PR updaten.

Reply:
Ich denke diese Frage sollten wir nächsten Montagmorgen beantworten können.